### ISSUE TITLE
Implement the most basic randomized world possible

### ISSUE THREAD

**Issue by Oleksii-Kshenskyi** [Oleksii-Kshenskyi/btagsil#32]
Think through and design the architecture of the world with:
- A couple rooms;
- A couple items in the rooms;
- A couple things to do with said items.

The catch: the world has to be random. You get a different world each time.

**Comment by Oleksii-Kshenskyi** [comment]
This is blocked by #40.

**Comment by Oleksii-Kshenskyi** [comment]
This is blocked by #41.

**Comment by Oleksii-Kshenskyi** [comment]
It's impossible for me to write a random world generation mechanism when I don't even know how anything in the world works yet. This will have to be closed as a "wontfix" although I'd created it myself.

Instead (in tune with the main "amnesiac god" theme of the game), let's research how possible the mechanic of "creating rooms" is. So instead of generating a pre-existing world, let's start with no rooms at all (the player is located in The Void), and give player the freedom to create rooms with their own hands, using a set of templates. The player could also create connections between rooms if they wanted to.
